### PR TITLE
New version: pizzaboxer.Bloxstrap version 2.7.0

### PR DESCRIPTION
**Pull request by pizzaboxer**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162150)

### CODE REVIEW

File: manifests/p/pizzaboxer/Bloxstrap/2.7.0/pizzaboxer.Bloxstrap.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: pizzaboxer.Bloxstrap
+PackageVersion: 2.7.0
+PackageLocale: en-US
+Publisher: pizzaboxer
+PublisherUrl: https://pizzaboxer.xyz
+PublisherSupportUrl: https://github.com/pizzaboxer/bloxstrap/issues
+PackageName: Bloxstrap
+PackageUrl: https://github.com/pizzaboxer/bloxstrap
```
Comment (sitiom) [inline manifests/p/pizzaboxer/Bloxstrap/2.7.0/pizzaboxer.Bloxstrap.locale.en-US.yaml:11]: Point to the new website:
```suggestion
PackageUrl: https://bloxstrap.pizzaboxer.xyz/
```

**Review by sitiom — CHANGES_REQUESTED** [review]

File: manifests/p/pizzaboxer/Bloxstrap/2.7.0/pizzaboxer.Bloxstrap.locale.en-US.yaml (hunk 1 above)
Comment (pizzaboxer) [reply manifests/p/pizzaboxer/Bloxstrap/2.7.0/pizzaboxer.Bloxstrap.locale.en-US.yaml:11]: Oh thanks, didn't catch that.

Comment (sitiom) [reply manifests/p/pizzaboxer/Bloxstrap/2.7.0/pizzaboxer.Bloxstrap.locale.en-US.yaml]: @pizzaboxer offtopic, but can you give me contributor role on the Bloxstrap server lol

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Comment by ImMejor35** [comment]
When I search bloxstrap via this command "winget search bloxstrap" it shows version 2.6.1?

File: manifests/p/pizzaboxer/Bloxstrap/2.7.0/pizzaboxer.Bloxstrap.locale.en-US.yaml (hunk 1 above)
Comment (pizzaboxer) [reply manifests/p/pizzaboxer/Bloxstrap/2.7.0/pizzaboxer.Bloxstrap.locale.en-US.yaml]: @sitiom Didn't know you were in the Discord server, what's your username?

Comment (sitiom) [reply manifests/p/pizzaboxer/Bloxstrap/2.7.0/pizzaboxer.Bloxstrap.locale.en-US.yaml]: Same username: @sitiom

**Pull request merged**